db_stress_tool/expected_state.cc

```diff
@@ -264,12 +285,17 @@ Status FileExpectedStateManager::Open() {
 
   std::string expected_state_file_path =
       GetPathForFilename(kLatestBasename + kStateFilenameSuffix);
+  std::string expected_persisted_seqno_file_path = GetPathForFilename(
+      kPersistedSeqnoBasename + kPersistedSeqnoFilenameSuffix);
   bool found = false;
   if (s.ok()) {
     Status exists_status = Env::Default()->FileExists(expected_state_file_path);
     if (exists_status.ok()) {
       found = true;
     } else if (exists_status.IsNotFound()) {
+      assert(Env::Default()
+                 ->FileExists(expected_persisted_seqno_file_path)
+                 .IsNotFound());
       found = false;
```

Comment:
nit: why do we set `found` to `false` again here when that is the default value

Reply:
For better literal meaning and prevent. I don't have a preference so fixed